x-pack/metricbeat/module/aws/s3_request/s3_request.go

```diff
@@ -216,5 +216,6 @@ func createS3RequestEvents(outputs []cloudwatch.MetricDataResult, regionName str
 
 	resultMetricSetFields.Put("bucket.name", bucketName)
 	event.MetricSetFields = resultMetricSetFields
+	event.RootFields.Put("aws.s3.bucket.name", bucketName)
```

Comment:
Is this field documented?

Reply:
I just added this field into fields.yml, thanks!